src/app/core/http-services/item-navigation.service.ts

```diff
@@ -6,6 +6,11 @@ import { bestAttemptFromResults } from 'src/app/shared/helpers/attempts';
 import { isRouteWithSelfAttempt, FullItemRoute } from 'src/app/shared/routing/item-route';
 import { appConfig } from 'src/app/shared/helpers/config';
 import { isASkill, isSkill, ItemType, ItemTypeCategory } from 'src/app/shared/helpers/item-type';
+import { decodeSnakeCase } from 'src/app/shared/operators/decode';
+import { pipe } from 'fp-ts/lib/function';
```

Comment:
```suggestion
import { pipe } from 'fp-ts/function';
```

Reply:
What's the diff?

Reply:
https://github.com/France-ioi/AlgoreaFrontend/pull/455#issuecomment-852823351

Reply:
Conversation to resolve after reading